src/spec.c

```diff
@@ -149,15 +157,43 @@ int isRdbLoading(RedisModuleCtx *ctx) {
   return isLoading == 1;
 }
 
-static void IndexSpec_TimedOut(RedisModuleCtx *ctx, void *data) {
-  IndexSpec *sp = data;
+static void IndexSpec_TimedOutProc(RedisModuleCtx *ctx, IndexSpec *sp) {
   // we need to delete the spec from the specDict, as far as the user see it,
   // this spec was deleted and its memory will be freed in a background thread.
+  RedisModule_Log(NULL, "notice", "Freeing index %s by timer", sp->name);
```

Comment:
This will spam the log file, temporary indexes are supposed to expire a lot

Reply:
I'll move it under `#ifdef _DEBUG`.